python/cuml/benchmark/runners.py

```diff
@@ -185,6 +266,77 @@ def _run_one_size(
             **cuml_param_overrides
         )
 
+    def _run_one_size_fil(
```

Comment:
I don't think we need this function two times.

Reply:
The first ` _run_one_size_fil` function only returns time whereas the second `_run_one_size_fil` returns both accuracy and time. This is the same as other algorithm classes.